src/code/graph.c

```diff
@@ -1,4 +1,3 @@
-#include "prevent_bss_reordering.h"
```

Comment:
Why did I have to remove this? The world may never know. BSS truly works in mysterious ways.

Reply:
bss reordering is a huge headache. There's a small explanation of how it works (or at least what we know about it) in that file (`prevent_bss_reordering.h`).
Most of the time either adding or removing that file as an include helps with the reordering, but not always, and that's the real hell